lib/promscrape/scraper.go

```diff
@@ -75,6 +76,9 @@ func Stop() {
 var (
 	globalStopCh chan struct{}
 	scraperWG    sync.WaitGroup
+	// UnReadyScrapeConfigs - zero value means, that
+	// all scrapeConfigs are inited and ready for work.
+	UnReadyScrapeConfigs int32
```

Comment:
Probably it would be better renaming it to something like `PendingScrapeConfigs`?

Reply:
Agreed, renamed.